RELEASE.md

```diff
@@ -13,8 +13,14 @@ Publishing the helm chart is currently manual, and we should move to an automate
 - [ ] Run the `./hack/generate-changelog.sh` script to generate the a draft changelog
 - [ ] Update the CHANGELOG.md file with the generated changelog
 - [ ] Commit, Tag and Push to Gitlab
-- [ ] Switch to the `gh-pages` branch and move the newly generated package to the `stable` helm repo
-- [ ] While on the `gh-pages` branch, run the `./build-index.sh` script to rebuild the indices for each repo
-- [ ] Commit and push the `gh-pages` branch to GitHub
-- [ ] Wait for the [CI job associated with your tag] (https://gitlab.com/nvidia/kubernetes/device-plugin/-/pipelines) to complete
-- [ ] Create a [new release](https://github.com/NVIDIA/k8s-device-plugin/releases) on Github with the changelog
+- [ ] Wait for the `Release` GitHub Action to complete
+- [ ] Create a [new release](https://github.com/NVIDIA/k8s-device-plugin/releases) from the draft release created by the GitHub Action
+- [ ] Wait for the `Publish Helm Chart` GitHub Action to complete
+
+## Troubleshooting
+- If the `Release` GitHub Action fails:
+    - check the logs for the error first.
+    - Manually run `./hack/package-helm-charts.sh` and updload the generated charts to the release assets
```

Comment:
Should we list the assets?

Reply:
is just 2 files with `tgz` sufix. I am not against it. up to you